manager.go

```diff
@@ -172,20 +169,17 @@ func (mgr *Manager) boot() error {
 		go process(mgr, i)
 	}
 
+	termFunc()
 	return nil
 }
 
 // Run starts processing jobs.
 // This method does not return unless an error is encountered while starting.
 func (mgr *Manager) Run() error {
-	err := mgr.boot()
-	if err != nil {
-		return err
-	}
-	for {
```

Comment:
I think this for loop would never return, I guess thats ok if it goes through the terminate path that exits though